Jenkinsfile

```diff
@@ -15,6 +15,7 @@ pipeline {
     GITHUB_CHECK_ITS_NAME = 'Integration Tests'
     ITS_PIPELINE = 'apm-integration-tests-selector-mbp/master'
     MAVEN_CONFIG = '-Dmaven.repo.local=.m2'
+    MAVEN_OPTS = '-Dmaven.ext.class.path=.mvn/opentelemetry-maven-extension.jar'
```

Comment:
This is the alternative to do not change anything in the pipeline related to the maven run wrapper